bitcoin/src/blockdata/transaction.rs

```diff
@@ -53,21 +53,29 @@ impl_hashencode!(Txid);
 impl_hashencode!(Wtxid);
 
 impl Txid {
-    /// The "all zeros" TXID.
+    /// The `Txid` used in a coinbase prevout.
     ///
```

Comment:
Would be nice if the doc specifically mentioned `txid` field on `OutPoint`

Reply:
Added a link to `OutPoint::COINBASE_PREVOUT`.